index.js

```diff
@@ -0,0 +1,61 @@
+// eslint-disable-next-line no-unused-vars 
+function validatePassword(password) { 
+
+  function length(password) {
+    if(password.length < 8) {
+      return false
+    }
+    else {
+      return true 
+    }
+  }
+  function upperCase(password){ 
+    if (password.toUpperCase() != password) {
+  
+      return true
+    }
+    else {
+      return false
+    }
+  }
+  function LowerCase(password){
+    if (password.toLowerCase() != password) 
+    {
+      return true
+    }
+    else
+    {
+      return false
+    }
+  }
```

Comment:
Testing the entire string like this won't work. Take for instance the valid password `P455w0rd!`. When you called `toLowerCase()` or `toUpperCase()` on it, it won't equal itself.

`P455w0rd!`.toLowerCase() is `p455w0rd!` which isn't equal to the original.